client_app/src/components/admin/AddShelterForm.js

```diff
@@ -0,0 +1,230 @@
+import React, { useState } from 'react';
+import './AddShelterForm.css';
```

Comment:
I think the path needs to be fixed, I'm getting errors when I run the client.
<img width="1690" alt="Screenshot 2025-03-04 at 11 34 27 AM" src="https://github.com/user-attachments/assets/f714ab91-2c0c-4108-a501-3ef1def8a56c" />